src/Init/Tactics.lean

```diff
@@ -1088,6 +1088,13 @@ Currently, all of these are on by default.
 -/
 syntax (name := omega) "omega" (config)? : tactic
 
+/--
+`bv_omega` is `omega` with an additional preprocessor that turns statements about `BitVec` into statements about `Nat`.
+Currently the preprocessor is implemented as `try simp only [bv_toNat] at *`.
+`bv_toNat` is a `@[simp]` attribute that you can (cautiously) add to more theorems.
+-/
+macro "bv_omega" : tactic => `(tactic| (try simp only [bv_toNat] at *) <;> omega)
```

Comment:
Instead of `try`, this could use `{ config := { failIfUnchanged := false }}`. Is there any reason to prefer the latter over the former? It’s more verbose, but it seems less roundabout than failing first and then recovering. (Maybe it simply doesn’t matter.)